### DISCUSSION TITLE
Is there any flag to start torchserve and load all models with dashboard startup

### DISCUSSION THREAD

**Discussion by abhinav-cashify** [cceyda/torchserve-dashboard#20] [General]
I m using this dashboard in production, I have dockerized this with torchserve on my end. But I can't find any flag in the dashboard to start torchserve and load all models from the model store (flags are provided by torchserve to do this).

Currently, I have to start torchserve and load models manually through the dashboard.

I need this flag so that I can automatically spin up the whole process in any event of failure.

**Comment by cceyda** [comment]
you can pass the `--init` flag introduced in the latest v0.6 release like `torchserve-dashboard --init` to start torchserve at dashboard initialization.
Although you still have to load the dashboard screen once for it to actually start!
this is because how cli params are passed through streamlit to torchserve command. I might have to do some refactoring of the code logic later but I haven't found a straightforward way to do it yet. 😢 

you can pick which models should be loaded at the start using the `torchserve.properties` file as described [here](https://pytorch.org/serve/configuration.html#load-models-at-startup)

I'll consider some dockerized use cases. I have mostly been using this on simple server setups for testing.

**Reply by abhinav-cashify** [reply]
Okay, I will try to do this on my end since it's on priority on my end for production.
I will raise a PR once its is done.
